domains/transfer/Transfer.API/API/Transfer/Api/Controllers/TransferAgreementProposalController.cs

```diff
@@ -6,39 +6,40 @@
 using Asp.Versioning;
 using DataContext.Models;
 using EnergyOrigin.ActivityLog.DataContext;
-using EnergyOrigin.TokenValidation.Utilities;
-using EnergyOrigin.TokenValidation.Values;
+using EnergyOrigin.TokenValidation.b2c;
 using FluentValidation;
 using FluentValidation.AspNetCore;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 
 namespace API.Transfer.Api.Controllers;
 
-[Authorize]
 [ApiController]
-[ApiVersion(ApiVersions.Version20240103)]
+[Authorize(Policy.Frontend)]
```

Comment:
Did we forget to migrate this aswell ? :S

Reply:
No. This is just how git shows diffs when you:

have two files named A and B.
Delete A
Rename B to A
Then it looks like what you see now.